Add meeting notes for 2019-10-29

klappradla opened the pull request:
Here's the notes for today's meeting. Please check if there's anything wrong or missing (and feel free to edit).

klappradla committed:
Add meeting notes for 2019-10-29

PragTob commented on meetings/2019-10-29.md:
```diff
@@ -0,0 +1,95 @@
+  - works best for first workshop
+  - quick vote: we start with intermediate
+- "intermediate" means confident with with computers
+  - this of course includes juniors
```
I'd add that we'd also take more experienced people that just haven't done elixir before (Mid/Senior+)

klappradla replied:
Added a second note for that.

PragTob commented on meetings/2019-10-29.md:
```diff
@@ -0,0 +1,95 @@
+- do an intermediate workshop with the snake tutorial
+  - write tutorial for attendees setting up snake game from scratch (using `scenic_new`)
+  - **Max, Nico and Markus** meet on Monday 4th to start with the guide (flash out structure, write first chapter)
+- **Tobi** is doing a general intro guide to Elixir (what's necessary for the snake tutorial)
```
I didn't explicitly say it, but unless someone needs it earlier I'd likely do this in December as I probably won't have a project in december aka lots of time. I might start earlier though :)

klappradla replied:
Added a note for that.

PragTob reviewed (commented):
Looking good to me, only small comments :)

![IMG_20180317_110254](https://user-images.githubusercontent.com/606517/67815980-c5b60180-faa8-11e9-8c6d-d14307488a32.jpg)

jvf commented on meetings/2019-10-29.md:
```diff
@@ -0,0 +1,95 @@
+- Markus
+- Max
+- Tobi
+- Daniel
```
Should be Gui (@ggpasqualino), not Daniel

klappradla replied:
Haha, oh my god 🙈 So sorry @ggpasqualino 🙇 (I actually don't even know why I came up with that in the first place...)

jvf commented on meetings/2019-10-29.md:
```diff
@@ -0,0 +1,95 @@
+### Which content
+
+- recap:
+  - nerves project (we stopped looking at it)
```
"stopped looking at it" seems wrong, we always talked about it but we did not have any further input and work done on it.

klappradla replied:
👍 rephrased it.

jvf commented on meetings/2019-10-29.md:
```diff
@@ -0,0 +1,95 @@
+  - snake does not work for finding a job, but valid for tutorial
+  - hardware project: makes sense for the future, but too much orga effort for now
+  - elixir girls app: we would need to update the guide (no ETS, less JS)
+    - pro: great for lots of elixir and no phoenix
+    - pro: has lots of the language features in it
```
Is it possible that these two points are referring to the Snake Tutorial?

klappradla replied:
In theory yes, but they were mentioned when referring to the Slack clone tutorial.

klappradla committed:
Fix typo in name :see_no_evil:

So sorry @ggpasqualino!

klappradla committed:
Update statement on nerves project

klappradla committed:
Add rough timeline to Tobi's task

klappradla committed:
Add note about audience

Add note that the tutorial with also suite people new to Elixir but with
some experience in other languages.

nickgnd reviewed (approved):
👏
